lib/importer/factory/object_factory.rb

```diff
@@ -0,0 +1,132 @@
+require 'importer/log_subscriber'
+module Importer
+  module Factory
+    class ObjectFactory
+      extend ActiveModel::Callbacks
+      define_model_callbacks :save, :create
+      after_save :attach_files
+
+      class_attribute :klass, :attach_files_service, :system_identifier_field
+
+      attr_reader :attributes, :files_directory, :object, :files
+
+      def initialize(attributes, files_dir = nil, files = [])
+        @files_directory = files_dir
+        @files = files
+        @attributes = attributes
+      end
+
+      # rubocop:disable Metrics/MethodLength
```

Comment:
I have a strong distaste for inline Rubocop exclusions. Could we move these to either .rubocop.yml or .rubocop_todo.yml?